upgrade/1.2/scripts/upgrade/create_rgw_buckets.sh

```diff
@@ -23,7 +23,34 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 
+function ssh_keygen_keyscan() {
```

Comment:
Mitch had this https://github.com/Cray-HPE/docs-csm/blob/e2820098e2146a055cdd9b6b463be6421eba9ff5/upgrade/1.2/scripts/upgrade/ncn-upgrade-common.sh#L88 looks like it is same? similar? FYI

Reply:
yeah -- identical.  i pirated it into the script that gets scp'd to s001 and runs ansible from there...